app.rb

```diff
@@ -1,9 +1,41 @@
 require 'sinatra/base'
+require_relative './lib/peep'
 
 class Chitter < Sinatra::Base
+  require 'sinatra/reloader' if development?
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  enable :sessions
+  attr_reader :peep_date
+
   get '/test' do
```

Comment:
It's good to remove any extraneous code from your implementation before making a Pull Request - there'll be lots of testing and exploration as you work on something, but it tidies it up to remove anything that is no longer needed for the program to meet its specification.